RDKEMW-5868 Native build workflow added for coverity

varatharajan568 opened the pull request:
Reason for change: Updated native-full-build workflow to remove subtec flag and add GITHUB_TOKEN
Test Procedure: None

Risks: Low
Priority: P1

varatharajan568 committed:
RDKEMW-5868 Native build workflow added for coverity

Reason for change: Updated native-full-build workflow to remove subtec flag and add GITHUB_TOKEN
Test Procedure: None

Risks: Low
Priority: P1

Signed-off-by: Varatharajan_Narayanan <Varatharajan_Narayanan@comcast.com>

varatharajan568 committed:
RDKEMW-5868 Native build workflow added for coverity

Reason for change: Updated native-full-build workflow to remove subtec flag and add GITHUB_TOKEN
Test Procedure: None

Risks: Low
Priority: P1

Signed-off-by: Varatharajan_Narayanan <Varatharajan_Narayanan@comcast.com>

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/middleware-player-interface/15/rdkcentral/middleware-player-interface

* Commit: a4461f020d1fb00c8313b4155a22697bd13f53bb

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/248071d49b50e874bf097dc07d4e2396)'

varatharajan568 committed:
Update native-full-build.yml

Updating subtec component to resolve the native build failure

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/middleware-player-interface/15/rdkcentral/middleware-player-interface

* Commit: df139be19dd4fd1592d72913fecd1e0670771e28

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/df1fa32e8cec0057320dc4bd215822e1)'

varatharajan568 committed:
Update native-full-build.yml

Added command to install sudo

varatharajan568 committed:
Update native-full-build.yml

Enabled the container env

varatharajan568 committed:
Update native-full-build.yml

installed sudo

varatharajan568 committed:
Update native-full-build.yml

Removed the meson & installed the latest version to resolve the error faced due to older version

balasaraswathy-n committed:
Merge branch 'develop' into feature/RDK-5868_1
